src/main/java/org/zaproxy/zap/extension/jwt/ui/JWTOptionsPanel.java

```diff
@@ -67,6 +66,17 @@
     private JTextField trustStoreFileChooserTextField;
     private JCheckBox enableClientConfigurationScanCheckBox;
 
+    /** JWT Fuzzer Options * */
+    private JPasswordField jwtHMacSignatureKey;
```

Comment:
These don't really seem to go together, maybe the field is a subset, or maybe the comment was in anticipation of something more?

Reply:
Actually this comment was written to specify that the below configurations of options panel is used in fuzzer and above onces are active scanner configurations.